sacred/dependencies.py

```diff
@@ -401,7 +401,7 @@ def iterate_sys_modules():
 def get_sources_from_modules(module_iterator, base_path):
     sources = set()
     for modname, mod in module_iterator:
-        if not getattr(mod, '__file__', None):
+        if not hasattr(mod, '__file__'):
```

Comment:
I think using `getattr` was introduced in https://github.com/IDSIA/sacred/commit/a6996a201a96c7d7c4aa84cf9af77f5fbfde9fb3 for being able to use pybind. But it seems its worth to keep a comment in the code, why this should be done.